src/components/RoomView/RoomView.jsx

```diff
@@ -29,10 +29,10 @@ const RoomView = () => {
   const navigate = useNavigate();
 
   // frontend state
-  const roomId = useSelector((state) => state.player.roomId);
```

Comment:
I did some clean up here to make all of the stuff imported from state.player into one line.